### ISSUE TITLE
Incorrect "Did you mean..." output with Hub

### ISSUE THREAD

**Issue by shadowfacts** [github/hub#923]
For example, when running `git puhs` Git will output
```
git: 'puhs' is not a git command. See 'git --help'.

Did you mean this?
	push
```
But when running `git frok` Git will output
```
git: 'frok' is not a git command. See 'git --help'.

Did you mean this?
	fsck
```
Completely ignoring the fact that there is a git subcommand called `fork`.

**Comment by mislav** [comment]
Good catch. Git is not aware that there is a hub command called `fork` because the `hub-fork` doesn't exist as an executable on the filesystem. It's unfortunate, but I don't have an idea how to fix it until we can split hub into individual executables.

**Comment by shadowfacts** [comment]
Is splitting hub into individual executables planned?

**Comment by mislav** [comment]
> Is splitting hub into individual executables planned?

Yes, it's something we've been thinking about for the next major version. It might be more trouble than it's worth, so we'll see.
